### PR TITLE
docs: Update Home Page and Setup Guides

### PR DESCRIPTION
**Pull request by prcutler**
Hi,

These are my first commits to Mainsail's docs, so if I didn't do something right or you don't like the changes, I won't be offended! :) Just let me know what you would like changed and I'm happy to help.

I've gone through the home page and the setup guide for edits.  As I'm getting used to the tone and style of the docs, I just start by reading and updating what's there.  

Generally the way I've been taught is think about translations and accessibility when writing docs.  I try to limit contractions (don't vs do not), use simpler language to make it easier for translation, and limit parentheses (). 

I'll keep reviewing and sending some edits in and then I'll start helping write some of the newer chapters for the next release.

Thanks for reviewing and let me know the feedback for what I should change.

### CODE REVIEW

File: docs/configurations/necessary-cfg.md [hunk 1]
```diff
@@ -22,28 +22,28 @@ The following configuration elements are optional, including making tweaks and a
...
 ## Display Status
-This is required for messages in your status panel, if you don't have a `[display]` in your configuration.
+`Display Status` is required for messages in your status panel, if you don't have `[display]` in your configuration.
 ```yaml
 [display_status]
 ```
 
 ## Pause, Resume, Cancel
-This enables pause / resume in mainsail.
+These macros control the current print job.
```
Comment (meteyou) [inline docs/configurations/necessary-cfg.md:38]: `[pause_resume]`is only to enable the PAUSE/RESUME commands in klipper...

**Comment by prcutler** [comment]
Hi,

I've reviewed and edited all of the Update guides.  Thank you to whomever wrote the documentation, it's always easier to edit than create!

I've tried to improve the spelling and grammar to improve readability.  I've also tried to remove contractions (don't vs do not), shorthand (config vs configuration), file names, etc. to make it easier for translators.

If there any mistakes or if anything doesn't make sense, please let me know and I'm happy to fix them.  I can take the feedback.  :)

File: docs/index.md [hunk 2]
```diff
@@ -17,30 +17,30 @@ Mainsail aims to make Klipper more accessible to end users through a lightweight
...
 - File Manager (gcode and config files)
 - File Editor (gcode and config files)
-- Print History / Stats
+- Print History and Statistics
 - Temperature Presets (with custom gcode)
 - Bed Mesh Leveling
 - Multi-Webcam support
-- Control power devices (relays, TPLink, Tasmota, ..)
-- Customizable UI (logos, backgrounds & even custom css)
+- Control power devices such as relays, TPLink and Tasmota devices, and more)
```
Comment (lixxbox) [inline docs/index.md:29]: remove ) at the end

File: docs/index.md [hunk 3]
```diff
@@ -17,30 +17,30 @@ Mainsail aims to make Klipper more accessible to end users through a lightweight
...
+- Customizable user interface includinglogos, backgrounds, and custom CSS
```
Comment (lixxbox) [inline docs/index.md:30]: includinglogos -> missing whitespace

File: docs/update-guide/klipper.md [hunk 4]
```diff
@@ -8,16 +8,18 @@ permalink: /update/klipper
...
+To update Klipper directly from its repository, run the following command in a terminal:
+
 ```bash
 cd ~/klipper
 git pull
 ```
 
-Restart Klipper (`sudo service klipper restart`) and check the `klippy.log`, if Klipper starts
-correctly and then continue the guide.
+Restart Klipper using the command `sudo service klipper restart` and check the `klippy.log`.  if Klipper startsed correctly without any errors you may continue.
```
Comment (lixxbox) [inline docs/update-guide/klipper.md:19]: typo: startsed

**Comment by prcutler** [comment]
Thanks!  Sorry I missed some of these.  I'll get the changes in by tomorrow.

**Review by lixxbox — CHANGES_REQUESTED** [review]
I made two small comments. if they are fixed, i think it can be merged.
Enabling the update manager via the mainsail editor is something we can do later.

**Review by lixxbox — APPROVED** [review]

**Pull request merged**